segway/run.py

```diff
@@ -4201,10 +4209,62 @@ def parse_options(argv):
     return task_spec, options, archives, traindir, annotatedir
 
 
+def check_gmtk_version():
+    """ Checks if the supported minimum GMTK version is installed """
+    # Typical expected output from "gmtkPrint -version":
+    # gmtkPrint (GMTK) 1.4.3
+    # Mercurial id: 8995e40101d2 tip
+    # checkin date: Fri Oct 30 10:51:44 2015 -0700
+
+    # Older versions:
+    # GMTK 1.4.0 (Mercurial id: bdf2718cc6ce tip checkin date: Thu Jun 25 12:31:56 2015 -0700)
+
+    # Try to open gmtkPrint to get the version
+    try:
+        # blocks until finished
+        output_string = subprocess.check_output(["gmtkPrint", "-version"])
+    # If GMTK was not found
+    except OSError:
+        # Raise a runtime error stating that GMTK was not found on the path
+        raise RuntimeError("GMTK cannot be found on your PATH.\nPlease "
+                           "install GMTK from "
+                           "http://melodi.ee.washington.edu/gmtk/ "
+                           "before running Segway.")
+
+    output_lines = output_string.splitlines()
+
+    # Check if there's only one line of output (for older versions)
+    if len(output_lines) == 1:
+        version_word_index = 1
+    else:
+        version_word_index = -1
+
+    # Get the first line of output
+    first_output_line = output_lines[0].decode()
+
+    # Get the version string from the proper word on the line
+    current_version_string = first_output_line.split()[version_word_index]
+
+    # Get the version number to compare with the minimum version
+    current_version = map(int, current_version_string.split("."))
+    version_zip = zip(current_version, MINIMUM_GMTK_VERSION)
```

Comment:
Can you use [`packaging.version`](https://packaging.pypa.io/en/latest/version/) instead? It should be installed with `setuptools`.